src/integration-test/tests/citizen/claim/steps/eligibility.ts

```diff
@@ -12,7 +12,7 @@ export class EligibilitySteps {
   complete (): void {
     I.amOnCitizenAppPage('/eligibility')
 
-    I.see('Find out if you can use this service')
+    I.see('Find out if you can make a claim using this service')
```

Comment:
Is that wording expected? Could not find it in prototype.

Reply:
Yes spoke to Ronan and Tarik yesterday and it's in the prototype

Reply:
Herrs the link to the prototype https://hmcts-cmc-prototype.herokuapp.com/prototypes/prototype-may-2018/dashboard/new-claim/eligibility/index